README.md

```diff
@@ -7,7 +7,19 @@ software licenses that are associated with each dependency in the project.
 The offline cache allows spandx to perform a truly airgap friendly scan of software
 projects.
 
-![badge](https://github.com/mokhan/spandx/workflows/ci/badge.svg)
+### Supported project types
+
+Spandx can work with following language's package managers. It utilises lock files generated by package managers to find dependencies.
+
+| Language  | Package Manager    | Tested in |
+| ------------ | --------------- | -------:|
+| Ruby         | bundler         | 1.17.3  |
+| Js           | Npm             | 6.13.4  |
+| Js           | Yarn            | 1.19.1  |
+| Python       | Pypi            | <>      |
```

Comment:
For Python we support 'Pipfile.lock' which is tied to the `pipenv` package manager at the moment. We've tested with version `6` of the `pipfile-spec`.